protocol/avdl/keybase1/teams.avdl

```diff
@@ -18,6 +18,14 @@ protocol teams {
       KID encKID;
   }
 
+  record TeamMembers {
+    array<string> owners;
+    array<string> admins;
+    array<string> writers;
+    array<string> readers;
```

Comment:
This is just for display to the user, and based on the doc here:

https://keybase.io/ccsandbox/team-internal-mvp